webapp/static/scripts/controller.js

```diff
@@ -1,297 +1,54 @@
-    W = window.innerWidth - 5;
-    H = window.innerHeight - 70;
-    canvas.width = W;
-    canvas.height = H;
-}
-
-function loop() {
-    ctx.clearRect(0, 0, W, H);
-    controller.draw();
-    let args = getArgs();
-    // establish a base case when there is no input event
-    if (moving[0] || moving[1]) {
-        if (args[0] != prevArgs[0] || args[1] != prevArgs[1] || args[2] != prevArgs[2]) {
-            if (enableSockets)
-                socket.emit('remoteOut', args);
-            else
-                console.log('remoteOut', args);
-        }
-        prevArgs = args;
-        window.requestAnimationFrame(loop);
-    }
-    else { // no input: set output data to idle
-        if (enableSockets)
-            socket.emit('remoteOut', [0, 0, 0]);
-        else
-            console.log('remoteOut', [0, 0, 0]);
-        prevArgs = [0, 0, 0];
-    }
-}
-
-function resize() {
-    updateDimensions();
-    ctx.clearRect(0, 0, W, H);
-    controller.draw();
-}
-
-function initRemote() {
-    canvas = document.getElementById("canvas");
-    canvas.addEventListener('touchstart', touchStart, false);
-    canvas.addEventListener('touchmove', touchMove, false);
-    canvas.addEventListener('touchend', touchEnd, false);
-    canvas.addEventListener('mousedown', mouseStart, false);
-    canvas.addEventListener('mouseup', mouseEnd, false);
-    canvas.addEventListener('mousemove', mouseMove, false);
-    ctx = canvas.getContext("2d");
-    updateDimensions(); // needs to be called for instantiating the Control class
-    controller = new Control();
-    window.addEventListener("gamepadconnected", function (e) {
-        console.log("Gamepad connected at index %d: %s. %d buttons, %d axes.",
-            e.gamepad.index, e.gamepad.id,
-            e.gamepad.buttons.length, e.gamepad.axes.length);
-    });
-    window.addEventListener("gamepaddisconnected", function (e) {
-        console.log("Gamepad disconnected from index %d: %s",
-            e.gamepad.index, e.gamepad.id);
-    });
-    window.addEventListener('resize', resize);// when window is resized
-    window.requestAnimationFrame(loop);
+var speedController = document.getElementById('slider-speed-control');
+var turnController = document.getElementById('slider-turning-control');
 
-    window.setInterval(getAxis, 16); // because gamepads aren't handled with events
-}
+var speedOMeter = document.getElementById('speed-o-meter');
+var turnOMeter = document.getElementById('turn-o-meter');
 
-// Controller object for use on canvas element
-class Control {
-    constructor() {
-        this.joystick = {
-            x: 0, y: 0, radius: 0,
-            color: "grey",
-            stick: {
-                x: 0, y: 0, radius: 0,
-                color: "#008B8B",
-                angle: 0,
-                touchRadius: 0
-            },
-            draw: function () {
-                this.stick.angle = Math.atan2((this.stick.y - this.y), (this.stick.x - this.x));
+const enableSockets = true;
```

Comment:
do we really need this?

Reply:
Yeah you're right. I used it for testing purposes of the PR but since websockets is working surprisingly well, I'll remove it